nxcomp/src/Makefile.am

```diff
@@ -121,25 +121,25 @@ libXcomp_la_LIBADD =					\
     @JPEG_LIBS@						\
     @PNG_LIBS@						\
     @Z_LIBS@						\
-    @PTHREAD_LIBS@					\
```

Comment:
Did you remove that because we already have `@PTHREAD_LIBS` in `..._LDFLAGS`?

I tend do agree with that, because `LIBADD` is only meant to be used with flags like `-L` and `-l`, while `LDFLAGS` can contain any linker flag. We don't really know what `AX_PTHREAD` will spit out as `PTHREAD_LIBS`, so it's probably safer to use `LDFLAGS` unless we'd knew that it will only ever contain `LIBADD`-supported linker flags (even though that's currently the case, as said, but it's not guaranteed that it won't change later on).